public/views/answers.html

```diff
@@ -1,89 +1,208 @@
 <div ng-cloak id="cards-container" ng-hide="game.state === 'game ended' || game.state ==='game dissolved'">
-  <div id="social-bar-container">
-    <player></player>
-  </div>
-  <div id="cards" class='hand'>
-
-    <span ng-show="winnerPicked()"></span>
-
-    <div ng-style="{'cursor':'pointer'}" class="card smallest" id='answers' ng-click="pickCard(card)" ng-hide="showTable || isCzar()"
-    ng-repeat="card in game.players[game.playerIndex].hand" ng-animate="{enter:'animated fadeInUp', leave: 'animated fadeInOut'}">
-        <span ng-bind-html-unsafe="card.text"></span>
-        <span id='selection-number' ng-show="showFirst(card)"> 1 </span>
-        <span id='selection-number' ng-show="showSecond(card)"> 2 </span>
+    <div id="social-bar-container">
+        <player></player>
     </div>
+    <div id="cards" class='hand'>
 
-    <span ng-repeat="answer in game.table">
-        <div class="card smallest" ng-style="pointerCursorStyle()" id='table' ng-click="pickWinning(answer)" ng-show="showTable || isCzar()"
-        ng-repeat="card in answer.card" ng-animate="{enter:'animated bounceInLeft'}" style="background-color:{{winningColor($parent.$index)}}">
+        <span ng-show="winnerPicked()"></span>
+
+        <div ng-style="{'cursor':'pointer'}" class="card smallest" id='answers' ng-click="pickCard(card)" ng-hide="showTable || isCzar() || game.state==='waiting for czar to draw a card'" ng-repeat="card in game.players[game.playerIndex].hand" ng-animate="{enter:'animated fadeInUp', leave: 'animated fadeInOut'}">
```

Comment:
HTML tag with many attributes can be written as:
```html
<div
  ng-style="{'cursor':'pointer'}"
  class="card smallest"
  id="answers"
  ...
  ng-repeat="card in game.players[game.playerIndex].hand"
  ng-animate="{enter:'animated fadeInUp', leave: 'animated fadeInOut'}"
>
  <span ng-bind-html-unsafe="card.text"></span>
  ...
</div>
```
Also, html attributes use dounble quotes instead of single quotes. `id` attribute has single quote